AppServices/Validations/UpdateCustomerValidator.cs

```diff
@@ -0,0 +1,115 @@
+﻿using APIDesafioWarren.Models;
+using Application.Models.DTOs;
+using FluentValidation;
+using FluentValidation.Validators;
+using System;
+
+namespace AppServices.Validations
+{
+    public class UpdateCustomerValidator : AbstractValidator<UpdateCustomerRequest>
+    {
+        public UpdateCustomerValidator()
+        {
+            RuleFor(c => c.FullName)
+                .NotEmpty()
+                .Must(v => v.ValidFullName())
+                .MaximumLength(300)
+                .MinimumLength(6);
+
+            RuleFor(c => c.Email)
+                .NotEmpty()
+                .MinimumLength(6)
+                .MaximumLength(254)
+                .EmailAddress(EmailValidationMode.Net4xRegex);
+
+            RuleFor(c => c.Cpf)
+                .NotEmpty()
+                .Must(ValidCPF)
+                .MinimumLength(11)
+                .MaximumLength(14);
+
+            RuleFor(c => c.Country)
+                .NotEmpty()
+                .Must(v => v.AllCharacteresArentEqualsToTheFirstCharacter())
+                .MaximumLength(35)
+                .MinimumLength(5);
+
+            RuleFor(c => c.City)
+                .NotEmpty()
+                .Must(v => v.AllCharacteresArentEqualsToTheFirstCharacter())
+                .MaximumLength(40)
+                .MinimumLength(3);
+
+            RuleFor(c => c.Cellphone)
+               .NotEmpty()
+               .WithMessage("Please complete this field")
+               .Must(v => v.AllCharacteresArentEqualsToTheFirstCharacter())
+               .MinimumLength(10)
+               .WithMessage("Invalid cellphone")
+               .MaximumLength(11)
+               .WithMessage("Invalid cellphone");
+
+            RuleFor(c => c.Birthdate)
+                .Must(ValidBirthDate)
+                .WithMessage("You must be at least 16 years old");
+
+            RuleFor(c => c.PostalCode)
+                .NotEmpty()
+                .Must(v => v.IsValidNumber());
+
+            RuleFor(c => c.Address)
+             .NotEmpty()
+             .Must(v => v.AllCharacteresArentEqualsToTheFirstCharacter())
+             .WithMessage("Please complete this field");
+
+            RuleFor(c => c.Number)
+                .NotEmpty()
+                .WithMessage("Please complete this field");
```

Comment:
```suggestion
            RuleFor(c => c.Number)
                .NotEmpty();
```
Sugiro que adicione uma validação para que o número for maior que 0